app/views/participants/_participant_info.slim

```diff
@@ -0,0 +1,54 @@
+ul.participant-info
+
+  - if @participant.biography && @participant.biography != ""
+    li.participant-info-category
+      | Biografia
+    li.participant-info-item
+      = @participant.biography
+
+  li.participant-info-category
+    | Email
+  li.participant-info-item
+    = @participant.email
+
+  li.participant-info-category
+    | Username
+  li.participant-info-item
+    = @participant.username
+
+  li.participant-info-category
+    | Sexo
+  li.participant-info-item
+    = @participant.sex_to_s
+
+  - if @participant.birthday != nil
+    li.participant-info-category
+      | Data de nascimento
+    li.participant-info-item
+      = @participant.birthday.strftime("%Y-%m-%d")
+
+  - if @participant.is_univ_student
+    - if @participant.minho_univ_student_id
+      li.participant-info-category
+        | Nº de Aluno UMinho
+      li.participant-info-item
+        = @participant.minho_univ_student_id
+
+    li.participant-info-category
+      | Curso
+    li.participant-info-item
+      - if @participant.is_inf_eng_student_at_minho_univ
```

Comment:
@frmendes Exactly my reaction when I saw it for the first time haha
I agree that it is better to create a method on the Participant model as you recommended. Will get on with it ;)